### ISSUE TITLE
Explain how to run traffic-light.udon

### ISSUE THREAD

**Issue by rmariani** [urbit/hoon-workbook#2]
The user is never given any indication of how to run the code. It's apparently not an app, so I can't figure it out, either.

https://github.com/urbit/hoon-workbook/blob/rob-edits/traffic-light.udon

Could you add some clear instructions for running the code, @baudtack ?

**Comment by rmariani** [comment]
I think the code should be rewritten as an app.

**Comment by baudtack** [comment]
This example is a dojo script. The intent is to illustrate state machines in hoon. It could run just as well as a generator,  just needs a little clean up for that.

**Comment by rmariani** [comment]
I think it being re-written as a generator would be good. It should always be clear how to readers how the code is supposed to be run.

**Comment by baudtack** [comment]
Agreed I'll take care of it. It should run just fine as a naked generator. With maybe a little fiddling.
